agent/handlers/v2/response_test.go

```diff
@@ -336,6 +337,7 @@ func TestTaskResponseMarshal(t *testing.T) {
 		"DesiredStatus":    "RUNNING",
 		"KnownStatus":      "RUNNING",
 		"AvailabilityZone": availabilityZone,
+		"VpcId":            vpcID,
```

Comment:
nit: is this `VpcId` missed from revision?

Reply:
We are still using `VpcId` in the JSON format of `TaskResponse`. I did so because of two reasons - 
1. `VpcId` is used as a key for VPC ID field in response from `aws ec2 describe-instances` CLI command.
2. All keys in `TaskResponse` JSON are PascalCase and `VPCID` was sticking out IMO.